labels.toml

```diff
@@ -0,0 +1,49 @@
+[Bug]
```

Comment:
Are those issue labels used by zenhub?

Reply:
Actually it's just a way of putting the GitHub issue labels under version control. It's not automated (yet), all I did was
```bash
labels fetch -o ComPWA -r tensorwaves
```
plus some token settings etc. (which is the hard part to automate). For more info see [`labels`](https://pypi.org/project/labels/).

Reply:
Ok